keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -389,6 +396,8 @@ This access mode will be enforced in two places:
 
 #### Scheduler Enforcement
 
+##### Alpha
+
 First is at the time a pod is scheduled. When scheduling a pod, if another pod
 is found using the same PVC and the PVC uses ReadWriteOncePod, then scheduling
 will fail and the pod will be considered UnschedulableAndUnresolvable.
```

Comment:
Below it says that the logic takes place in PreFilter, but it's actually implemented as Filter. Please update.

Reply:
For alpha, the scheduler enforcement was implemented in the [`PreFilter`](https://github.com/kubernetes/kubernetes/blob/v1.25.2/pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go#L120) phase. Separate checks are performed in [`Filter`](https://github.com/kubernetes/kubernetes/blob/v1.25.2/pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go#L171) that are unrelated to ReadWriteOncePod.

For beta this enforcement will be moved to the `Filter` phase.

Reply:
ahhhh, sorry about that. That's because the decision is global to the cluster, rather than being per node, right?

Then disregard my comment about caching results in Filter :)